test/runSvcGenerateTests.py

```diff
@@ -328,6 +328,10 @@ def test_genResetFiles(self):
 
 	def test_genExternalSig(self):
 		out = "genExternalSigLog.txt"
+
+		if OPENSSL:
+			command(['echo' , '"TEST NOT RAN, OPENSSL BUILS DO NOT HAVE THIS FEATURE"' ], out, False)
```

Comment:
it seems mean to pick on spelling in a test case, but builds has a D :P

Reply:
(if you choose to fix this, feel free to just fix up the patch and then merge the series without going through another round of reviews!)

Reply:
Thank you. If you have not noticed yet, my spelling is horrible. I actually really appreciate these comments, its really embarrassing to see silly typos in one of the packaged releases... Software cannot look like it was made by humans, any sign of human error shows weakness! Linus said that once...right?